softdevice_controller/CHANGELOG.rst

```diff
@@ -21,6 +21,8 @@ Added
   See the :c:func:`sdc_hci_cmd_vs_qos_channel_survey_enable` function.
 * Support for starting the scanner without setting scan parameters.
   Previously the controller would assert (DRGN-17623).
+* Vendor-specific HCI command to enable utilization of remote APR on the local TX power when using LE Power Control.
+  See :c:func:`sdc_hci_cmd_vs_set_power_control_apr_handling` (DRGN-17355).
 
 Changes
```

Comment:
@seko-nordic , should we mention HCI API changes?

Reply:
it does not affect NCS integration, so may be not.